x-pack/qa/sql/src/main/resources/string-functions.sql-spec

```diff
@@ -93,9 +93,10 @@ SELECT "first_name" orig, REPEAT("first_name",2) reps FROM "test_emp" WHERE ASCI
 selectInsertWithLcase
 SELECT "first_name" orig, INSERT("first_name",2,1000,LCASE("first_name")) modified FROM "test_emp" WHERE ASCII("first_name")=65 ORDER BY "first_name" ASC LIMIT 10;
 
-// AWAITS FIX for https://github.com/elastic/elasticsearch/issues/32589
+// related to for https://github.com/elastic/elasticsearch/issues/32589
+// H2 is Locale sensitive, while ES-SQL is not (so far)
 // selectInsertWithLcaseAndLengthWithOrderBy
-//SELECT "first_name" origFN, "last_name" origLN, INSERT(UCASE("first_name"),LENGTH("first_name")+1,123,LCASE("last_name")) modified FROM "test_emp" WHERE ASCII("first_name")=65 ORDER BY "first_name" ASC, "last_name" ASC LIMIT 10;
+// SELECT "first_name" origFN, "last_name" origLN, INSERT(UCASE("first_name"),LENGTH("first_name")+1,123,LCASE("last_name")) modified FROM "test_emp" WHERE ASCII("first_name")=65 ORDER BY "first_name" ASC, "last_name" ASC LIMIT 10;
```

Comment:
I think instead of commenting them out we could move them into a separate sql-spec file and we can add a logic that will not add them to the list of test [here](https://github.com/elastic/elasticsearch/blob/edf83c1d87951e350be26ce86764ef783f80ab8a/x-pack/qa/sql/src/main/java/org/elasticsearch/xpack/qa/sql/jdbc/SqlSpecTestCase.java#L37) for locales where H2 is messed up.

Reply:
Yep. Makes sense. I'll make the change.